components/formatter/PodImages.vue

```diff
@@ -34,6 +47,10 @@ export default {
 <template>
   <span>
     <span>{{ images[0] }}</span><br>
-    <span v-if="images.length-1>0" class="plus-more">{{ t('generic.plusMore', {n:images.length-1}) }}</span>
+    <span
+      v-if="images.length-1>0"
+      v-tooltip.bottom="imageLabels"
+      class="plus-more"
```

Comment:
Can we add `cursor: help` to the style of `plus-more` - so when you hover over it the cursor changes - gives a little feedback to the user.